.github/workflows/docker.yaml

```diff
@@ -89,6 +84,79 @@ jobs:
             type=raw,value=${{ github.event.pull_request.head.ref }}
             type=raw,value=stable,enable=${{ startsWith(github.event.ref, 'refs/tags/v') }}
 
+      - name: Build and push livepeer docker image
+        uses: docker/build-push-action@v4
+        with:
+          build-args: |
+            BUILD_TAGS=${{ steps.build-tag.outputs.build-tags }}
+          # linux/arm64 is disabled until
```

Comment:
The related ticket is resolved, should we remove this comment?

Reply:
you're right. updated.